django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def bind_to_form(self, form, name):
```

Comment:
I can see that `BoundField.__init__()` may be changed in the future, but `bind_to_form()` itself should just be an operation to get a bound field out of a regular one.
I can't see where that would need additional arguments. Do you have something in mind?